client/src/components/Login.js

```diff
@@ -61,18 +65,57 @@ function Login(props) {
   const classes = useStyles();
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [openSnack, setOpenSnack] = useState(false);
+  const [snackText, setSnackText] = useState("");
   const value = useContext(UserContext);
+  const vertical = "top";
+  const horizontal = "center";
+
+  const handleSnack = (props) => {
```

Comment:
Since this is just within a single component, let's use 'message' or 'text' or something instead of props